pkg/logger/handler_test.go

```diff
@@ -89,3 +89,41 @@ func TestLogger(t *testing.T) {
 	// get logResponse
 	<-responseChan
 }
+
+func TestBadResponse(t *testing.T) {
+
+	g := gomega.NewGomegaWithT(t)
+
+	predictorRequest := []byte(`{"instances":[[0,0,0]]}`)
+	predictorResponse := "BadRequest\n"
```

Comment:
The case of tensorflow-model-server that returns 400 with json which can be logged is not captured by this plain text response. We can try forcing an application-json content-type like in theofpa/kfserving@9330ec7af9036915e985aa8743a387173aa4827d